app/src/main/java/com/it3048/mobiledeviceproject/MeetingAdapter.kt

```diff
@@ -6,22 +6,23 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
+import com.it3048.mobiledeviceproject.dto.Meeting
 
-class MeetingAdapter(val c:Context, val meetingList:ArrayList<MeetingDAO>):RecyclerView.Adapter<MeetingAdapter.meetingViewHolder>() {
-    inner class meetingViewHolder(val v:View):RecyclerView.ViewHolder(v){
-        val title = v.findViewById<TextView>(R.id.meetingNameTxt)
-        val date = v.findViewById<TextView>(R.id.meetingDateTxt)
-        val description = v.findViewById<TextView>(R.id.meetingDescriptionTxt)
-        val link = v.findViewById<TextView>(R.id.meetingUrlTxt)
+class MeetingAdapter(val c:Context, val meetingList: ArrayList<Meeting>):RecyclerView.Adapter<MeetingAdapter.MeetingViewHolder>() {
+    inner class MeetingViewHolder(v:View):RecyclerView.ViewHolder(v){
```

Comment:
Capitalized the first letter of the inner class to fit with Kotlin coding conventions.

Source:
https://kotlinlang.org/docs/coding-conventions.html

Reply:
Good.